package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "onfido-sdk-ui",
-  "version": "0.11.1",
+  "version": "0.12.0-prerelease1",
```

Comment:
might be better use  semver for our pre releases, this would be a release candidate so something like `"0.12.0-rc1"`.
see: http://semver.org/